arclightning_backend/src/main.rs

```diff
@@ -0,0 +1,201 @@
+extern crate futures;
+extern crate hyper;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+extern crate toml;
+
+use futures::future;
+
+use hyper::rt::{Future, Stream};
+use hyper::service::service_fn;
+use hyper::{Body, Method, Request, Response, Server, StatusCode};
+
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::Read;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+
+// after this concept is further understood, will switch to 'Either'
+type ResponseFuture = Box<Future<Item = Response<Body>, Error = hyper::Error> + Send>;
+
+// using PartialEq for unit tests
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
```

Comment:
Can you move the derived traits line one more line down? Since both comments reference this line.